.github/workflows/workflows.yaml

```diff
@@ -0,0 +1,141 @@
+      
+      - name: Build and Push docker image (AMD64)
+        id: docker
+        uses: tignis/docker-github-action@v2.3.1
+        with:
+          images: ${{ inputs.images }}
+          acr-username: ${{ secrets.acr-username }}
+          acr-password: ${{ secrets.acr-password }}
+          acr-registry-url: ${{ inputs.acr-registry-url }}
+          pip-extra-index-url: ${{ secrets.pip-extra-index-url }}
+          push: ${{ inputs.push }}
+          docker-build-context: ${{ inputs.docker-build-context }}
+          dockerfile: ${{ inputs.dockerfile }}
+          platforms: 'linux/amd64'
+          tag-prefix: 'amd64-'
+          GITHUB_TOKEN: ${{ inputs.GITHUB_TOKEN }}
+          
+      - name: Extract clean tag
+        id: clean-tag
+        run: |
+          TAG=${{ steps.docker.outputs.tag }}
+          CLEAN_TAG=${TAG#*:amd64-}
+          echo "value=${CLEAN_TAG}" >> $GITHUB_OUTPUT
+        shell: bash
+
+      - name: Get image digest
+        id: digest
+        run: |
+          echo "digest=$(docker inspect ${{ steps.docker.outputs.tag }} --format='{{index .RepoDigests 0}}' | cut -d'@' -f2)" >> $GITHUB_OUTPUT
+        shell: bash
+
+  docker-arm64:
+    runs-on: [self-hosted, linux, ARM64]
+    outputs:
+      tag: ${{ steps.docker.outputs.tag }}
+      digest: ${{ steps.digest.outputs.digest }}
+      image: ${{ inputs.images }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      
+      - name: Build and Push docker image (ARM64)
+        id: docker
+        uses: tignis/docker-github-action@v2.3.1
+        with:
+          images: ${{ inputs.images }}
+          acr-username: ${{ secrets.acr-username }}
+          acr-password: ${{ secrets.acr-password }}
+          acr-registry-url: ${{ inputs.acr-registry-url }}
+          pip-extra-index-url: ${{ secrets.pip-extra-index-url }}
+          push: ${{ inputs.push }}
+          docker-build-context: ${{ inputs.docker-build-context }}
+          dockerfile: ${{ inputs.dockerfile }}
+          platforms: 'linux/arm64'
+          tag-prefix: 'arm64-'
+          GITHUB_TOKEN: ${{ inputs.GITHUB_TOKEN }}
+
+      - name: Get image digest
+        id: digest
+        run: |
+          echo "digest=$(docker inspect ${{ steps.docker.outputs.tag }} --format='{{index .RepoDigests 0}}' | cut -d'@' -f2)" >> $GITHUB_OUTPUT
+        shell: bash
+
+  docker-manifest:
+    needs: [docker-amd64, docker-arm64]
+    runs-on: ubuntu-latest
+    outputs:
+      tag: ${{ steps.manifest.outputs.tag }}
+    steps:
+      - name: Login to ACR
+        uses: docker/login-action@v3
+        with:
+          registry: ${{ inputs.acr-registry-url }}
+          username: ${{ secrets.acr-username }}
+          password: ${{ secrets.acr-password }}
+      
+      - name: Build manifest
+        shell: bash
+        run: |
+          TAG=${{ needs.docker-amd64.outputs.tag }}
+          TAG=${TAG#${{ needs.docker-amd64.outputs.image }}:${{ needs.docker-amd64.outputs.tag-prefix }}}
```

Comment:
There might be a way to ask the `docker/build-push-action` to output the image tag, then we don't need to do this weird manipulation.